award_emojis.go

```diff
@@ -56,38 +56,47 @@ const (
 	awardSnippets     = "snippets"
 )
 
-// ListEmojiAwardsOptions represents the available options for listing emoji
+// ListAwardEmojiOptions represents the available options for listing emoji
 // for each resources
 //
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/award_emoji.html
-type ListEmojiAwardsOptions ListOptions
+type ListAwardEmojiOptions ListOptions
+
+// CreateAwardEmojiOptions represents the available options for awarding emoji
+// for a resource
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/award_emoji.html#award-a-new-emoji
+type CreateAwardEmojiOptions struct {
+	Name string `json:"name"`
+}
```

Comment:
Looks good! But please place this declaration above the first `Create...` method.

Reply:
Updated, PTAL